# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1549302593 0
#      Mon Feb 04 17:49:53 2019 +0000
# Branch stable
# Node ID af47f245a2ada829fa87f339bbe8e7c64a189bda
# Parent  df42ea23502f185358798894a38205c3251673f2
bump version for first 5.1 release candidate

* configure.ac (AC_INIT): Set version to 5.0.91.
(OCTAVE_PATCH_VERSION): Now 91.
(OCTAVE_RELEASE_DATE): Set to 2019-02-04.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -15,38 +15,38 @@ dnl Copyright (C) 1993-2019 John W. Eato
 ### GNU General Public License for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <https://www.gnu.org/licenses/>.
 
 ### Initialize Autoconf
 AC_PREREQ([2.65])
-AC_INIT([GNU Octave], [5.0.90], [https://octave.org/bugs.html], [octave],
+AC_INIT([GNU Octave], [5.0.91], [https://octave.org/bugs.html], [octave],
         [https://www.gnu.org/software/octave/])
 
 ### Declare version numbers
 
 dnl Note that the version number is duplicated here and in AC_INIT because
 dnl AC_INIT requires it to be static, not computed from shell variables.
 
 ## The description of the Octave version number in the etc/HACKING.md
 ## explains how to update these numbers for release and development
 ## versions.
 
 OCTAVE_MAJOR_VERSION=5
 OCTAVE_MINOR_VERSION=0
-OCTAVE_PATCH_VERSION=90
+OCTAVE_PATCH_VERSION=91
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION argument.
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2019 John W. Eaton and others."
 
-OCTAVE_RELEASE_DATE="2019-01-24"
+OCTAVE_RELEASE_DATE="2019-02-04"
 
 ## The "API version" is used as a way of checking that interfaces in the
 ## liboctave and libinterp libraries haven't changed in a backwardly
 ## incompatible way when loading .oct files.  A better way to do this is with
 ## library versioning, but not all systems support it.
 ##
 ## NOTE: This macro will be removed in a future version of Octave!
 ## If you insist on checking for features using a version number, use the
